packages/react-components/.storybook/preview.js

```diff
@@ -2,27 +2,6 @@ import * as rootPreview from '../../../.storybook/preview';
 import { FluentDocsContainer } from '../src/DocsComponents/FluentDocsContainer.stories';
 import { FluentDocsPage } from '../src/DocsComponents/FluentDocsPage.stories';
 
-/** @type {NonNullable<typeof rootPreview.parameters['options']>} */
-const options = {
-  storySort: {
-    method: 'alphabetical',
-    /**
-     * @see https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy#sorting-stories
-     */
-    order: [
-      'Concepts',
-      [
-        'Introduction',
-        'Developer',
-        ['Quick Start', 'Migrating from @fluentui᜵react v8', 'Styling Components', 'Positioning Components'],
-      ],
-      'Theme',
-      'Components',
-      'Migrations',
-    ],
-  },
-};
-
 /** @type {typeof rootPreview.decorators} */
 export const decorators = [...rootPreview.decorators];
```

Comment:
I don't think this is needed to enable that new functionality. the purpose to have non trivial options configuration as a separate constant was when we re-generate storybook configs we don't do AST. thus for contributor it's far easier to undo changes that are not related to new config - in this case the option object. 

can you please revert this  change?

Reply:
I'm not sure why, but the typings complain about having the options in a separate const. I'll see if there's another way I can work around the typing complaints.

Reply:
Maybe the issue is the NonNullable in the way the options were declared before?

Reply:
Seems this is no longer needed 🤷‍♀️  so we reverted it

Reply:
@Hotell i moved this back after realizing that typings weren't complaining, but rather storybook-build complained:
![d577ce94-4710-4157-ad80-479878923122](https://user-images.githubusercontent.com/1434956/158260031-40e024e1-fd71-4793-97be-79d86a104466.jpg)

It looks like storybook requires these settings to be inline for static analysis or something.

Reply:
^ this is interesting and indeed disturbing. like you cannot use valid javascript because there is some magic going on? what happened to you storybook ha ? 🥺

Reply:
I'm guessing that's part of the "magic" that enables code splitting, sort of like how with webpack you can't have dynamically-generated `import()` paths.